### ISSUE TITLE
Disabled Toggle Label does not announce state when using Talkback

### ISSUE THREAD

**Issue by daneuber** [microsoft/fluentui#21282]
### Environment Information

- **Package version(s)**: 8.49.5
- **Browser and OS versions**: Edge 97.0.1072.55 on Android 12 

### Describe the issue:
Toggle FluentUI Control does not provide 'disabled' information for Toggle label when navigated to on Android with Talkback

### Please provide a reproduction of the issue in a codepen:
1. Open Edge web browser on an Android device
2. Go to FluentUI Toggle control web page - https://developer.microsoft.com/en-us/fluentui#/controls/web/toggle
3. Start Talkback on the device
4. Swipe through until you get to the 'Disabled and checked' Toggle example

#### Actual behavior:
On the disabled toggle control label Talkback announces only the label content (in this case, 'disabled and checked')

#### Expected behavior:
Talkback should announce the label content and 'disabled' when screen reader focus is on it

### Documentation describing expected behavior
MAS 4.2.1

**Comment by gouttierre** [comment]
I’ve added this to our backlog for contributors as it’s low on our priority list and we’re likely not going to get to it for a while as we’re currently focused on larger coherence work items and enhancements to our performance.
